modules/hal_silabs/simplicity_sdk/config/ll/sl_btctrl_config.h

```diff
@@ -123,6 +123,10 @@
 #define SL_BLUETOOTH_NO_VS_HCI_COMMANDS
 #endif
 
+#ifdef CONFIG_SILABS_SISDK_RAIL_MULTIPROTOCOL
+#define SL_RAIL_LIB_MULTIPROTOCOL_SUPPORT 1
```

Comment:
This symbol seems to be set by the `rail_lib_(single|multi)protocol` components in SiSDK, and it is always explicitly set to `0` or `1` depending on which variant of the library is selected. Should this be set from CMakeLists.txt whenever RAIL is included, instead of in a Bluetooth-specific header? Other protocol stacks may depend on it too.